### PR TITLE
Update support for Gatsby 4

### PR DESCRIPTION
**Pull request by crummy**
This plugin blocks upgrades to Gatsby 4.0

### CODE REVIEW

**Comment by dukebody** [comment]
Same issue here.

File: package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "font-awesome"
   ],
   "peerDependencies": {
-    "gatsby": "^3.0.0",
+    "gatsby": "^4.0.0",
```
Comment (InfamousStarFox) [inline package.json:21]: To make sure backwards compatibility isn't broken with Gatsby v3, what about:
```
"gatsby": "^3.0.0 || ^4.0.0",
```

**Comment by InfamousStarFox** [comment]
As a work around for this plugin not working in Gatsby v4, you can use

```
import { config, dom } from "@fortawesome/fontawesome-svg-core";
config.autoAddCss = false;

--------------------

<Helmet>
    <style>{dom.css()}</style>
</Helmet>
```

**Comment by jzabala** [comment]
Thank you very much for the PR @crummy

**Pull request merged**